lib/ui/painting/image_decoder_impeller.cc

```diff
@@ -483,7 +526,11 @@ void ImageDecoderImpeller::Decode(fml::RefPtr<ImageDescriptor> descriptor,
                                                  gpu_disabled_switch]() {
           sk_sp<DlImage> image;
           std::string decode_error;
+#ifdef FML_OS_ANDROID
+          if (false) {
+#else
           if (context->GetCapabilities()->SupportsBufferToTextureBlits()) {
```

Comment:
Should this just be a different question for the capabilities?  "ShouldUseBufferToTextureBlits"?